spec/migrations/20230830134742_delete_nil_timestamp_event_streams_spec.rb

```diff
@@ -0,0 +1,16 @@
+require_migration
+
+describe DeleteNilTimestampEventStreams do
+  let(:event_stream_stub) { migration_stub(:EventStream) }
+
+  migration_context :up do
+    it "removes nil timestamp events, leaving others" do
+      good = event_stream_stub.create!(:timestamp => Time.now.utc)
+      event_stream_stub.create!(:timestamp => nil)
+
+      migrate
+
+      expect(event_stream_stub.pluck(:id, :timestamp).flatten).to eq([good.id, good.timestamp])
```

Comment:
This is failing on timestamp precision,
```
  1) DeleteNilTimestampEventStreams#up removes nil timestamp events, leaving others
     Failure/Error: expect(event_stream_stub.pluck(:id, :timestamp).flatten).to eq([good.id, good.timestamp])
     
       expected: [4, 2023-08-30 17:33:20.160235809 +0000]
            got: [4, 2023-08-30 17:33:20.160235000 +0000]
```

Reply:
ok, I was hoping to keep it simple.  I guess I can check that's not nil.

Reply:
You could use `.to be_within()` or `.to_i` both

Reply:
Thanks, I went with present since that's the intention of it.